src/Mod/Measure/Gui/TaskMeasure.cpp

```diff
@@ -163,6 +163,11 @@ TaskMeasure::TaskMeasure()
     // engage the selectionObserver
     attachSelection();
 
+    if (auto* doc = App::GetApplication().getActiveDocument()) {
+        m_deletedConnection = doc->signalDeletedObject.connect(
+            std::bind(&TaskMeasure::onObjectDeleted, this, std::placeholders::_1));
```

Comment:
The linter is proposing a slight simplification:
```suggestion
            [this](auto && obj) { onObjectDeleted(obj); });
```